### DISCUSSION TITLE
How to use the app?

### DISCUSSION THREAD

**Discussion by saschabrockel** [IgnisDa/ryot#674] [Q&A] [answered]
Hey, I found time to install the app via docker-compose but I have no idea of how to do anything.

I mean I can add for example movies manually but it's surely not the way you normally do it. I know I can use TMDB for that somehow but the documentation does not really help me to understand how everything works. I guess I'm just blind. I'm especially interested in the person feature now but I guess it's too new to be documented.

Can somebody please explain it?

**Comment by IgnisDa** [comment, marked as answer]
As of now, there is no way to search for people. I plan to work on that soon (#675). As a workaround, you can first search for a movie of the person you want to track.

Example:
Suppose you want to monitor Ryan Gosling. So you would search for "Drive" (or any other movie that he worked on) in the movies search tab. Then open the media. In the overview tab, you will find Ryan Gosling listed under "Actors". Click on that. THen in the new page that opens, you need to go the the actions tab and under "More actions", click on "monitor person". When there are any updates, you will be notified in your configured notification platforms.

As for doing tracking manually, you mostly wouldn't. Take a look at the [integration docs](https://ignisda.github.io/ryot/integrations.html) and set that up to sync progress automatically.

**Reply by IgnisDa** [reply]
LMK if you have any other questions.

**Reply by saschabrockel** [reply]
LOL so sorry. The search never worked for me somehow but now does...

**Reply by IgnisDa** [reply]
The search input has a debounce of 1 second. It is intentional so as to not spam the backend.

**Reply by saschabrockel** [reply]
Makes sense but I waited longer before. I guess my system was overloaded.
